src/Kusto/Kusto/Commands/GetAzureRmKustoCluster.cs

```diff
@@ -12,12 +12,14 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using System;
```

Comment:
Please remove unused usings from all your files. Otherwise it isn't clear what you actually depend on.